tests/DependencyInjection/Compiler/AdminMakerCompilerPassTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Tests\DependencyInjection;
+
+use PHPUnit\Framework\TestCase;
+use Prophecy\Argument;
+use Sonata\AdminBundle\DependencyInjection\Compiler\AdminMakerCompilerPass;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+
+/**
+ * @author Gaurav Singh Faujdar <faujdar@gmail.com>
+ */
+class AdminMakerCompilerPassTest extends TestCase
+{
+    private $adminMaker;
+
+    public function setUp()
+    {
+        if (5 == PHP_MAJOR_VERSION && !class_exists('Symfony\Component\Console\CommandLoader\CommandLoaderInterface')) {
```

Comment:
wasn't this `||`?

Reply:
I guess yes 🤔

Reply:
Yes it was ||

https://github.com/sonata-project/SonataAdminBundle/commit/e4d3fc0dae47e67d6d1ad4ca9fff9e038e6af01d